catalog/app/containers/Search/Route.ts

```diff
@@ -38,13 +38,13 @@ const PackageSearchParamsFromSearchParams = S.transform(
       resultType: SearchModel.ResultType.QuiltPackage as const,
       filter: S.decodeSync(Filter.PackageFilter.fromSearchParams)(params),
       userMetaFilters: S.decodeSync(UserMetaFilters.fromSearchParams)(params),
-      latestOnly: params.rev?.[0] === 'latest',
+      latestOnly: params.rev?.[0] !== 'all',
```

Comment:
"all" and "latest" are used as just strings literals everywhere, right? It makes sense, to use constants or enum to show all available options and prevent mistyping.

Reply:
there are no other available options, it's just a boolean flag which i've decided to encode as `rev=all` in querystring for future proofing (and for general aesthetics)